### PR TITLE
VER-4037 [DH] HLFE (FE) - Modify content for the DST mailbox page

### PR DESCRIPTION
**Pull request by davehill-tds**

### CODE REVIEW

File: app/uk/gov/hmrc/helplinefrontend/views/helplinesByService/Helpline.scala.html [hunk 1]
```diff
@@ -50,6 +49,13 @@
     @if(helpdesk != "dst"){
         @summaryList(rows = rows, classes = "govuk-summary-list--no-border")
     } else {
+@*        work here*@
```
Comment (adamliversidge) [inline app/uk/gov/hmrc/helplinefrontend/views/helplinesByService/Helpline.scala.html:52]: comment needs removing

File: app/uk/gov/hmrc/helplinefrontend/views/helplinesByService/Helpline.scala.html [hunk 2]
```diff
@@ -26,8 +26,7 @@
       h1: H1,
       h2: H2,
       summaryList: SummaryList,
-      insetText: InsetText)
-
+      insetText: InsetText, govukWarningText : GovukWarningText)
```
Comment (adamliversidge) [inline app/uk/gov/hmrc/helplinefrontend/views/helplinesByService/Helpline.scala.html:29]: new line would be good to keep the formatting

**Comment by davehill-tds** [comment]
Screenshot 2023-11-02 at 09 44 58

File: conf/messages [hunk 3]
```diff
@@ -358,6 +358,9 @@ helplines-search.voa.int=+44 300 050 1501
 helplines-search.voa.times=Monday to Friday: 8am to 6pm<br>Closed weekends and bank holidays.
 
 helplines-search.dst.h1=Contact the DST mailbox
+helplines-search.dst.p.1=Contact the DST mailbox to recover your sign in details for Digital Services Tax.
+helplines-search.dst.p.2=Digital Services Tax is a tax on large businesses (UK turnover over £25 million) operating in online marketplaces, search engines and social media.
+helplines-search.dst.warning=You can be fined up to £5,000 if you do not register.
```
Comment (adamliversidge) [inline conf/messages:363]: This should be as per the content in the prototype, the £5000 message is the example from the design system - ie it should be "This mailbox is not for Making Tax Digital or your personal tax account."

**Review by adamliversidge — APPROVED** [review]

**Pull request merged**